WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -101,7 +101,7 @@ class ReaderPostCardCell: UITableViewCell {
             static let likeButtonHint = NSLocalizedString("reader.post.button.like.accessibility.hint",
                                                           value: "Likes the post.",
                                                           comment: "Accessibility hint for the like button on the reader post card cell")
-            static let likedButtonHint = NSLocalizedString("reader.post.button.like.accessibility.hint",
+            static let likedButtonHint = NSLocalizedString("reader.post.button.liked.accessibility.hint",
```

Comment:
Love that our tooling picks these up :)

Next step, fail the build in the PR that introduced them. https://github.com/wordpress-mobile/release-toolkit/issues/446